app/src/hooks/api/useSearchApi.ts

```diff
@@ -19,8 +22,22 @@ const useSearchApi = (axios: AxiosInstance) => {
     return data;
   };
 
+ /**
+   * Get occurrence map data
+   *
+   * @param {Feature} [spatialSearch]
+   * @return {*}  {Promise<IGetMapOccurrenceData[]>}
+   */
+  const getMapOccurrenceData = async (spatialSearch?: Feature): Promise<IGetMapOccurrenceData[]> => {
+    axios.defaults.params = { spatial: JSON.stringify(spatialSearch) };
```

Comment:
I don't think we want to set params this way. Possibly this is actually altering the base axios instance that other calls might use. You should be able to pass params in as a second parameter to the `axios.get` function (exact syntax I cant remember).